tests/unittest/executor/test_rpc_proxy.py

```
@@ -55,28 +54,29 @@ def create_proxy(self, tp_size: int):
 
-    @pytest.mark.skip(reason="https://nvbugs/5579234")
-    @pytest.mark.parametrize("num_reqs", [1, 10])
+    @pytest.mark.parametrize("num_reqs", [1, 5, 10])
     def test_tp1(self, num_reqs):
         tokenizer = TransformersTokenizer.from_pretrained(model_path)
         prompt = "A B C D"
         prompt_token_ids = tokenizer.encode(prompt)
         max_tokens = 8
 
         with self.create_proxy(tp_size=1) as proxy:
+            logger_debug(f"[Test] Proxy created", color="green")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)